src/Beacon.jsx

```diff
@@ -199,20 +198,16 @@ export class Beacon extends Component {
   getHorizontalAttachment(bounds) {
     const marginRight = window.innerWidth - bounds.right;
     const marginLeft = bounds.left;
-    const tolerance = window.innerWidth / TOOLTIP_TOLERANCE_RATIO;
-    if (Math.abs(marginRight - marginLeft) < tolerance) {
-      return 'center';
-    } else if (marginRight < marginLeft) {
+    // const tolerance = window.innerWidth / TOOLTIP_TOLERANCE_RATIO;
+    if (marginLeft < TOOLTIP_HORIZONTAL_TOLERANCE) {
```

Comment:
I changed the logic here so it always centers the tooltips horizontally unless the target is right on the edge of the screen.